app/conversations/routes.py

```diff
@@ -145,3 +144,23 @@ def get_conversations():
     response = {"conversations": results}
 
     return jsonify(response), 200
+
+
+@bp.route("/conversation/<conversation_uuid>", methods=["GET"])
+@cross_origin()
+def get_conversation(conversation_uuid):
+    """
+    Get a single conversation.
+    """
+    session_uuid = request.headers.get("X-Session-Id")
+    session_uuid = validate_uuid(session_uuid, uuidType.SESSION)
+    check_uuid_in_db(session_uuid, uuidType.SESSION)
+
+    validate_uuid(conversation_uuid, uuidType.CONVERSATION)
+    check_uuid_in_db(conversation_uuid, uuidType.CONVERSATION)
+    response = build_single_conversation_response(conversation_uuid)
```

Comment:
I think it might be better to not have a helper function build the response unless this is done with the other endpoints. The reason I say this is because if we want to edit the response it would be good to have all the response code in the same place as the other endpoints. It also makes documenting the endpoint easier because wouldn’t have to change the documentation in multiple places.

Reply:
No change. I prefer cleaner endpoints.